src/coreComponents/physicsSolvers/simplePDE/LaplaceFEM.hpp

```diff
@@ -141,17 +142,15 @@ class LaplaceFEM : public SolverBase
   } laplaceFEMViewKeys;
   //END_SPHINX_INCLUDE_04
 
-protected:
-
-  virtual void PostProcessInput() override final;
-
 private:
 
   string m_fieldName;
   timeIntegrationOption m_timeIntegrationOption;
 
 };
 
+ENUM_STRINGS( LaplaceFEM::timeIntegrationOption, "SteadyState", "ImplicitTransient", "ExplicitTransient" )
```

Comment:
We can also assign strings to enums declared inside a class. The only caveat is the macro has to be called at the namespace scope.